code/include/playfab/PlayFabAuthenticationContext.h

```diff
@@ -40,3 +40,8 @@ namespace PlayFab
         bool IsEntityLoggedIn();
     };
 }
+
+struct PlayFabAuthenticationContextHolder
+{
+    std::shared_ptr<PlayFab::PlayFabAuthenticationContext> context;
+};
```

Comment:
1.  The idea is that we can indirectly give clients a handle to a ref counted object accross the C interface.  See PlayFabResultDuplicate/CloseHandle for examples of how this works.  XUser & XUserHandles in the GDK uses a very similar pattern.  That said, this particular class isn't actually needed because we don't expose the auth context publicly anymore (since I introduced the User class) so I can remove it.
2.  No problem, I can fix that.

---
In reply to: [590834852](https://github.com/jplafonta/XPlatCppSdk/pull/2#discussion_r590834852) [](ancestors = 590834852)